store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -49,86 +49,123 @@
-
-    private static final String O_UFAT = "ufat";
-
-    private static final String O_UIT = "uit";
+    private static final String O_ST    = "st";
+    private static final String O_VP    = "vp";
+    private static final String O_STP   = "stp";
+    private static final String O_BID   = "bid";
+    private static final String O_UFA   = "ufa";
+    private static final String O_UFAT  = "ufat";
+    private static final String O_UIT   = "uit";
 
     /** attributes for store type OpenIO **/
-    private static final String O_AP = "ap";
-    private static final String O_PP = "pp";
-    private static final String O_ACCOUNT = "acc";
-    private static final String O_NS = "ns";
-    private static final String O_URL = "url";
-    private static final String OPENIO = "OPENIO";
-
-    private static final String NOT_ALLOWED_INTERNAL = " is not allowed for internal storetype";
-    private static final String NOT_ALLOWED_EXTERNAL = " is not allowed for external storetype";
-    private static final String NOT_ALLOWED = " is not allowed for edit";
-    private static final String INVALID_STORE_TYPE = "invalid storetype";
-    private static final String HELP_EXTERNAL_NAME = "  only name can be edited for external store volumes ";
-    private static final String MISSING_ATTRS = " is missing";
-    private static final String NOT_ALLOWED_ID = "id cannot be specified when adding a volume";
-
-    private static final String H_STORE_TYPE = "Store type: internal or external";
-    private static final String H_STORAGE_TYPE = "Name of the store provider (S3, ObjectStore, OPENIO)";
-    private static final String H_BUCKET_ID = "S3 Bucket ID";
-    private static final String H_VOLUME_PREFIX = "Volume Prefix";
-    private static final String H_URL = "URL of OpenIO";
-    private static final String H_PROXY_PORT = "Proxy port";
-    private static final String H_ACCOUNT_PORT = "Account port";
-    private static final String H_ACCOUNT = "Name of account";
-    private static final String H_NAME_SPACE = "Namespace";
-    private static final String H_STORE_MANAGER_CLASS = "Optional parameter to specify non-default store manager class path";
-    private static final String H_USE_INFREQUENT_ACCESS_THRESHOLD = "Use Infrequent access storage class blob size threshold";
-    private static final String H_USE_INFREQUENT_ACCESS = "Use Infrequent access storage class - AWS only";
-    private static final String H_USE_INTELLIGENT_TIERING = "Use Intelligent tiering storage class - AWS only";
-    private static final String A_ID = "id";
-    private static final String A_TYPE = "type";
-    private static final String A_PATH = "path";
-    private static final String A_NAME = "name";
-    private static final String A_COMPRESS = "compress";
-    private static final String A_COMPRESS_THRESHOLD = "compressThreshold";
-    private static final String A_STORE_TYPE = "storeType";
-
+    private static final String O_AP    = "ap";
```

Comment:
We can separate out all constants into a constant file.

Reply:
Created new file as VolumeCLIConstants